ledgers/ccf/scripts/set_contract_enclave_check_attestation_flag.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import http
+import os
+import sys
+
+from ccf.clients import Identity
+from ccf.clients import CCFClient
+
+# pick up the logger used by the rest of CCF
+from loguru import logger as LOG
+
+## -----------------------------------------------------------------
+ContractHome = os.environ.get("PDO_HOME") or os.path.realpath("/opt/pdo")
+CCF_Keys = os.environ.get("PDO_LEDGER_KEY_ROOT") or os.path.join(ContractHome, "ccf", "keys")
```

Comment:
This can be cleaned up. ContractHome is not used and the keys should be in the ledger key root.

Reply:
line 30 is consistent with other variable definitions, see for example line 29. We use this pattern all over pdo. nothing unique about 30.

Reply:
Right, because it's old code that gets copy-pasted every time, but it's not used and never cleaned up.

Reply:
i think it is reasonable to have default values for variables. at best, this should be discussed as separate PR. the pattern is not restricted to ccf scripts.

Reply:
there is a lot of clean up to be done on these scripts to make them match expectations. suggest we not worry too much about optimization and clarity & focus this PR on flow and function.